core/src/main/java/org/eclipse/dash/licenses/IProxySettings.java

```diff
@@ -0,0 +1,29 @@
+/*************************************************************************
+ * Copyright (c) 2022 STMicroelectronics and others.
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution, and is available at https://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *************************************************************************/
+package org.eclipse.dash.licenses;
+
+import java.net.http.HttpClient;
+
+/**
+ * Injectable HTTP proxy settings.
+ */
+public interface IProxySettings {
+
+	/** The default port number ({@value}) for HTTP proxy. */
+	int DEFAULT_PROXY_PORT = 8080;
```

Comment:
That's right. This is a left-over from an earlier iteration of the contribution in which the components of the proxy server URL were configurable via mojo properties, but that didn't make sense because the `settings.xml` provides all that is required.